apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Query/UserRoleQuery.cs

```diff
@@ -36,4 +37,60 @@ public partial class Query
     public async Task<IEnumerable<UserRoleDto>> GetUserRolesAsync(
         [Service] IMarketParticipantClient_V1 client) =>
         await client.UserRolesGetAsync();
+
+    public async Task<IEnumerable<ActorViewDto>> GetUserRoleViewAsync(
+        Guid userId,
+        [Service] IMarketParticipantClient_V1 client)
+        {
+            var usersActors = await client
+            .UserActorsGetAsync(userId)
+            .ConfigureAwait(false);
+
+            var fetchedActors = new List<ActorDto>();
+
+            foreach (var actorId in usersActors.ActorIds)
+            {
+                fetchedActors.Add(await client.ActorGetAsync(actorId).ConfigureAwait(false));
+            }
```

Comment:
This will fetch actors in serial. Provided that the endpoint can handle it, you might want to consider fetching in parallel instead. See this [example](https://github.com/Energinet-DataHub/greenforce-frontend/blob/dce3725c58e02195150d592fbbc7d2e8d01d8e17/apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Resolvers/WholesaleResolvers.cs#L33) for how to do that using Task.WhenAll